e2e_playwright/basic_app_test.py

```diff
@@ -93,14 +93,14 @@ def test_check_total_websocket_message_number_and_size(page: Page, app_port: int
     # and expected
 
     # BackMsg's; currently: ~70 bytes
-    TOTAL_WEBSOCKET_SENT_SIZE_THRESHOLD_BYTES: Final = 150
+    TOTAL_WEBSOCKET_SENT_SIZE_THRESHOLD_BYTES: Final = 150  # noqa: N806
     # Number of websocket messages sent
-    EXPECTED_WEBSOCKET_MESSAGES_SENT: Final = 1
+    EXPECTED_WEBSOCKET_MESSAGES_SENT: Final = 1  # noqa: N806
 
     # ForwardMsg's; currently: ~1200 bytes
-    TOTAL_WEBSOCKET_RECEIVED_SIZE_THRESHOLD_BYTES: Final = 2000
+    TOTAL_WEBSOCKET_RECEIVED_SIZE_THRESHOLD_BYTES: Final = 2000  # noqa: N806
     # Number of websocket messages received
-    EXPECTED_WEBSOCKET_MESSAGES_RECEIVED: Final = 8
+    EXPECTED_WEBSOCKET_MESSAGES_RECEIVED: Final = 8  # noqa: N806
```

Comment:
I think its fine in this case to use an uppercase naming since those are constants specific to a test function that can be changed eventually. But it would make it more complicate to move these constants to the module level.